packages/graphback/src/index.ts

```diff
@@ -4,14 +4,11 @@
 export * from './GraphQLBackendCreator'
 export * from './IGraphQLBackend'
 
-// DDL layer
-export * from './database/migrations/DataResourcesManager'
-
 // Logger management
-export * from './utils/logger'
-export * from './database';
+// export * from './utils/logger'
 
 export * from '@graphback/codegen-client'
 export * from '@graphback/codegen-core'
```

Comment:
Since this package is used everywhere maybe it makes sense to call it core?

Reply:
Renamed to something like `@graphback/core`?

Reply:
yeah. This is just conversation. No strong opinion as it is used everywhere

Reply:
Yeah I agree with this. I believe I mentioned it somewhere before.